src/pluto/format/save.cr

```diff
@@ -0,0 +1,10 @@
+module Pluto::Format::Save
```

Comment:
Ditto as the `Open` module. Provides nice convenience method for saving an image to file.